src/models/StyleSheetManager.js

```diff
@@ -1,10 +1,14 @@
 // @flow
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import StyleSheet, { CONTEXT_KEY } from './StyleSheet'
+import StyleSheet from './StyleSheet'
 import ServerStyleSheet from './ServerStyleSheet'
 import { tagConstructorWithTarget } from './BrowserStyleSheet'
 
+// TODO: Import this from ./StyleSheet again
+// when we figured out why it is undefined during testing sometimes
+const CONTEXT_KEY = '__styled-components-stylesheet__'
```

Comment:
It's undefined because of a circular dependency. In #1501 I'm moving it to `src/constants.js`